### ISSUE TITLE
Iris Copilot text is visually appearing as heading but it is not programmatically defined

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5099]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

People who use screen reader for navigation will be impacted as screen reader is not announcing any heading information on the text which are visually appealing as heading. Many people prefer heading mode to navigate on the page and if user didn't find any heading, they might get confused.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://mcapshelpuat.microsoft.com/

### How to reproduce the issue?

Step 1: Open the URL: https://mcapshelpuat.microsoft.com/ in New Edge/Chrome browsers.
Step 2: 'MCAPSHelp' Home page will appear. 
Step 3: Navigate to the controls present in 'My help' section -> Invoke start control in 'Account Management Requests' Tile-> 'Iris Copilot' appears. 
Step 4: Navigate using heading short cut key 'H' in the page.
Step 5: Verify whether visually appearing heading texts defined in heading or not.

### What do you expect?

Visually appearing as heading text should be defined in heading tag.
Ex: Iris Copilot text should be defined in <H2> tag.

### What actually happened?

Visually appearing heading texts are not defined in heading.
Observation:
'Iris Copilot' text is not defined under heading programmatically.

### Do you have any screenshots or recordings to repro the issue?

<img width="1012" alt="MAS 1 3 1-Visually appearing heading texts are not defined in heading" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/a0f91eb1-e3ca-44a7-bad4-81fe19dc4e55">

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from MCAPS: [ISSUE 1767725](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767725)

**Comment by compulim** [comment]
This is out of scope of Web Chat. The title bar is done by the Iris Bot. Web Chat does not implement the title bar.

**Comment by Kirank915** [comment]
As per the above comment, this issue is out of scope and not under bot framework team. Hence closed the bug and partner team (WWL) also changed the ownership in bugs.
